shortcodes/ucf-faq-list-shortcode.php

```diff
@@ -157,11 +157,22 @@ public static function shortcode( $atts ) {
 				'topic_class'    => 'h5',
 			), $atts, 'ucf-faq-topic-list' );
 
-			$topics = get_terms( 'topic', array(
-				'post_type' => array( 'faq' )
-			) );
+			$args = array(
+				'post_type'      => 'faq',
+				'posts_per_page' => -1
+			);
+
+			$taxonomy = 'topic';
+			$terms = get_terms( $taxonomy, $args );
+
+			foreach ( $terms as $index=>$term ) {
+				$sort_order = get_field( 'topic_sort_order', $taxonomy.'_'.$terms[$index]->term_id );
+				$terms[$index]->sort_order = ( $sort_order === NULL ) ? "0" : $sort_order;
```

Comment:
It might be easier to read with the reverse logic:

```suggestion
				$terms[$index]->sort_order = ( $sort_order ) ? (int)$sort_order : 0;
```